webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -45,27 +45,35 @@ export function getRoomId(): string {
   return document.getElementById("room")!.dataset.roomId!;
 }
 
-function elementId(stream: MediaStream, type: "video" | "audio" | "feed") {
-  return `${type}-${stream.id}`;
+function elementId(streamId: string, type: "video" | "audio" | "feed") {
+  return `${type}-${streamId}`;
 }
 
 export function addVideoElement(
   stream: MediaStream,
   label: string,
-  mute: boolean = false
+  muted: boolean = false
 ): void {
-  const id = elementId(stream, "video");
-  let video = document.getElementById(id) as HTMLVideoElement;
-
-  if (!video) {
-    video = setupVideoFeed(stream, label);
+  const videoId = elementId(stream.id, "video");
+  const audioId = elementId(stream.id, "audio");
+  let video = document.getElementById(videoId) as HTMLVideoElement;
+  let audio = document.getElementById(audioId) as HTMLAudioElement;
```

Comment:
do we still use audio elements?

Reply:
I have introduced an audio element in the last two commits. It's needed to hear video tile that has not been displayed yet. Now we have structure as follows:
videoElement - to display video
audioElement - to play sound

If you ask me if it won't  get out of sync - I don't know but first tests are positive.

Reply:
Ok, any reason why not use video only and show/hide it when needed?

Reply:
Damn, I tested the version with videoElement only but the audio from videoElement is played only when videoElement also receives video frames so it can sync them and play together. This was the reason to introduce audioElements.

Reply:
Ok, so let's leave it for now. Possibly we can try using  https://stackoverflow.com/questions/6433900/syncing-html5-video-with-audio-playback